app/views/public_recipes/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1 class="text-2xl font-extrabold text-gray-900  md:text-3xl lg:text-4xl"> Public Recipes List </h1>
+<div class="flex justify-center">
+  <%= image_tag ("recipes4.jpg"), :class => "w-9/12 border rounded-3xl"%>
+</div>
+<div class="wrapper">
+
+<% if @public_recipes.length != 0 %>
+<% @public_recipes.each do |recipe| %>
+  <%= render partial: 'public_recipe_card', locals: { recipe: recipe, user_id: recipe.id } %>
```

Comment:
- [ ] As demanded by project requirements, clicking on a public recipe should lead to the details of the recipe. Kindly fix this, please.
![image](https://user-images.githubusercontent.com/94551484/219607797-e5ae10d0-a60e-466b-8be4-f4350225773c.png)
![image](https://user-images.githubusercontent.com/94551484/219607994-2dfc8514-b4c3-4b9e-83a6-3c9df293e1a7.png)

Reply:
Hello @Kamba56 
Thanks for your kind review and feedback (Click on recipe name, It will lead you to the recipe detail page)